notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Second sentence, suggested rewording: "This also helps us to confirm that by default, the lightkurve plotting function uses the sap_flux on the flux axis, rather than the detrended kspsap_flux; the sap_flux can be used for studies of stellar variability, but these signals need to be removed from the light curve for detailed studies of eclipses and transits."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>